tests/gtest/avif_fuzztest_helpers.cc

```diff
@@ -187,9 +187,20 @@ ::testing::Environment* SetEnv(const char* name, const char* value) {
 
 //------------------------------------------------------------------------------
 
+// Helper macros to ensure TEST_DATA_DIRS is a quoted string.
+// NOTE: TEST_DATA_DIRS MUST NOT be a quoted string before stringification
+// otherwise it will be quoted twice, resulting in the use of an incorrect path.
+#define AVIF_TO_STRING(S) #S
+#define AVIF_STRINGIFY(S) AVIF_TO_STRING(S)
```

Comment:
Could be in the `#ifdef TEST_DATA_DIRS` block below because it is used only there

Reply:
Done.